protocols/luci-proto-wireguard/luasrc/model/cbi/admin_network/proto_wireguard.lua

```diff
@@ -81,10 +81,22 @@ preshared_key = section:taboption(
             "cryptography for post-quantum resistance.")
 )
 preshared_key.password = true
-preshared_key.datatype = "and(base64,rangelength(44, 44))"
+preshared_key.datatype = "and(base64,rangelength(44,44))"
 preshared_key.optional = true
 
 
+fwmark = section:taboption(
+  "advanced",
+  Value,
+  "fwmark",
+  translate("Firewall Mark"),
+  translate("Optional. 32-bit hexadecimal mark for outgoing encrypted " ..
+            "packets. Enter value including leading <code>0x</code>.")
+)
+fwmark.datatype = "and(hexstring,rangelength(0,8))"
```

Comment:
Tracked it down to this expression which does not validate: `(val <= 4294967296)`

Throw in a small number like 113 and it will fail. I think it reserves a bit for the sign and uses 32bit signed ints. Even on a 64bit architecture. This also affects the `range()` function, but probably never caused trouble so far.

Ideas? I don't want to start bitshifting. Can we just stick to hex, including the 0x of course?

Context:
````
function fwmark(val)
	if val then
		if (val:match("^0x[a-fA-F0-9]+$") ~= nil) and (#val < 11) then
			return true
		end
		val = tonumber(val)
		--return ((val ~= nil) and (val >= 0) and (val <= 4294967296))
		return (val <= 4294967296)
	end
  return false
end
````